ci/pipelines/meta/deploy_meta.sh

```diff
@@ -256,6 +256,7 @@ set +x
 if [[ "${GEODE_FORK}" != "${UPSTREAM_FORK}" ]]; then
   echo "Disabling unnecessary jobs for forks."
   pauseJobs ${META_PIPELINE} set-reaper-pipeline
+  pauseJobs ${META_PIPELINE} set-mass-test-run-pipeline
```

Comment:
Please also add `enableFeature mass-test-run` to `ci/pipelines/meta/deploy_meta.sh` line ~285 just below where `pr` is enabled

Reply:
done